duckduckgo/macos-browser
# Use latest find in page

Task/Issue URL: 
 - https://app.asana.com/0/1200467065115223/1200979569868269/f
 - https://app.asana.com/0/1199178362774117/1200631417796340/f
 - https://app.asana.com/0/1200467065115223/1200825763313285/f

Tech Design URL: n/a
CC:

**Description**:
Update browser to use latest "find in page" with applied bugfixes and removed instrumentation.

**Steps to test this PR**:
1. Use "find in page"
1. Should no longer see pixel data in Kibana
2. Should not change spacing on hackernews
3. Should update highlights when window is resized

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
